packages/teleport/src/Player/ProgressBar/ProgressBar.tsx

```diff
@@ -55,8 +56,38 @@ export type ProgressBarProps = {
   toggle: () => void;
   style?: React.CSSProperties;
   id?: string;
+  playSpeed?: (speed: number) => void;
 };
 
+function PlaySpeedSelector(props: { onChange?: (speed: number) => void }) {
+  if (!props.onChange) {
+    return null;
+  }
+
+  let handleChange = (event: any) => {
```

Comment:
```suggestion
  const handleChange = (event: any) => {
```
also, that `any` should be typed, i think you can hover over onChange and see the type (depending on what IDE you use), i think it would be `React.ChangeEvent<HTMLSelectElement>`